test/run_tests.sh

```diff
@@ -80,7 +80,6 @@ esac
 # Permissions-related sanity checking
 if [[ "$USER_MODE" -eq "0" && $EUID -ne 0 ]]; then
    echo "This script must be run as root in order to install dependencies.  Use -u for user mode." 1>&2
-   exit 1
```

Comment:
Why not exit after this error message? Same comment below.

Reply:
Oh, right, this one was too much. Fixed. The other one is need, otherwise the script is never accepting `-u`.